.github/workflows/ci-e2e.yaml

```
@@ -14,6 +14,8 @@ on:
         description: 'Base url for backend API'
         required: true
         default: 'https://api.watch.kausal.tech/v1'
+      schedule:
+        - cron: '0 22 * * *' # 22:00 UTC, 01:00 EEST
```

Do we need to run this on weekends? Maybe it could just be Monday to Friday

```suggestion
      schedule:
        - cron: '0 22 * * MON-FRI' # 22:00 UTC, 01:00 EEST
```

---

Good point!